tox.ini

```diff
@@ -1,7 +0,0 @@
-[flake8]
```

Comment:
why was the `tox.ini` file deleted?

Reply:
Thanks @rgil-splunk for notifying this. It was deleted accidentally, now I have added the file again. If everything looks fine kindly approve the PR.